getting-started-knative/.gitignore

```diff
@@ -32,4 +32,6 @@ target/
 pom.xml.tag
 pom.xml.releaseBackup
 pom.xml.versionsBackup
-release.properties
\ No newline at end of file
+release.properties
+
+Dockerfile
```

Comment:
Why is the `Dockerfile` in `.gitignore` but still committed?

Reply:
done